File: bookingservice/src/main/java/com/microservice/bookingservice/model/ScheduleDate.java
package com.microservice.bookingservice.model;

import com.fasterxml.jackson.annotation.JsonBackReference;
import com.fasterxml.jackson.annotation.JsonManagedReference;


import javax.persistence.*;
import java.util.ArrayList;
import java.util.List;

@Entity
public class ScheduleDate {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String date;

    @OneToMany(mappedBy = "scheduleDate", cascade = CascadeType.ALL, orphanRemoval = true)
    @JsonManagedReference
    private List<Slot> slots = new ArrayList<>();

    @ManyToOne
    @JoinColumn(name = "schedule_id")
    @JsonBackReference
    private Schedule schedule;


    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public List<Slot> getSlots() {
        return slots;
    }

    public void setSlots(List<Slot> slots) {
        this.slots = slots;
    }

    public Schedule getSchedule() {
        return schedule;
    }

    public void setSchedule(Schedule schedule) {
        this.schedule = schedule;
    }
}

File: Accomodation_service/WebApplication1/Models/ScheduleDate.cs
using System;
using System.Collections.Generic;

namespace WebApplication1.Models
{
    public partial class ScheduleDate
    {
        public ScheduleDate()
        {
            Slots = new HashSet<Slot>();
        }

        public long Id { get; set; }
        public string? Date { get; set; }
        public long? ScheduleId { get; set; }

        public virtual Schedule? Schedule { get; set; }
        public virtual ICollection<Slot> Slots { get; set; }
    }
}
